pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBase.java

```diff
@@ -941,22 +941,27 @@ private OpBatchReceive<T> nextBatchReceive() {
         return opBatchReceive;
     }
 
-    protected final void notifyPendingBatchReceivedCallBack(OpBatchReceive<T> opBatchReceive) {
```

Comment:
Why did we change the method param signature? Maybe `OpBatchReceive` is more clear?